app/views/groups/_group_members.html.erb

```diff
@@ -1,24 +1,25 @@
 <% group.users.each do |user| %>
-  <div class="col-xs-12 col-sm-12 col-md-6 col-lg-4 groups-members-card-container">
-    <div class="groups-members-card">
+  <div class="<%= policy(group).admin_access? ? 'col-xs-12 col-sm-12 col-md-6 col-lg-4 groups-members-card-container' : 'col-xs-12 col-sm-12 col-md-6 col-lg-3 groups-members-card-container' %>">
```

Comment:
Consider moving this to a helper as this is being used multiple places

Reply:
Sure

Reply:
@tachyons Added the helpers